data/benefits/javascript/region-ile-de-france-aide-image-r-lyceens-labaz.yml

```
@@ -0,0 +1,19 @@
+label: aide image R pour les lycéens et les lycéennes
+prefix: l’
+institution: region_ile_de_france
+description: Une aide de 100 euros est proposée aux 500 000 lycéens et lycéennes d’Île-de-France afin de financer leur forfait Navigo imagine R scolaire 2024-2025, dont le coût annuel s'élève à 382,40 euros. Cette aide de 100 euros s'ajoute aux aides départementales.
+conditions:
+  - Télécharger l'application mobile LABAZ disponible sur les stores et vous créer un compte.
```

```suggestion
  - Télécharger l'application mobile LABAZ disponible sur Google Play / App Store et vous créer un compte.
```
On pourrait aller plus loin en mettant le lien de l'app.